chain-core/src/state/tendermint.rs

```diff
@@ -342,4 +343,10 @@ impl TendermintVotePower {
     pub fn zero() -> Self {
         TendermintVotePower(0)
     }
+
+    /// note TendermintVotePower::from(c).as_non_base_coin() != c if c had a fractional part
+    pub fn as_non_base_coin(self) -> Coin {
+        Coin::new(self.0 as u64)
```

Comment:
Isn't `TendermintVotePower` is `Coin / 10000_0000`

Reply:
A good catch -- yes, this was a bug

Reply:
I'll fix it